h11/_connection.py

```diff
@@ -1,17 +1,6 @@
 # This contains the main Connection class. Everything in h11 revolves around
 # this.
-from typing import (
-    Any,
-    Callable,
-    cast,
-    Dict,
-    List,
-    Optional,
-    overload,
-    Tuple,
-    Type,
-    Union,
-)
+from typing import Any, Callable, cast, Optional, overload, Union
```

Comment:
I believe the desire is to reduce merge conflicts by keeping these in separate lines.